src/datta_lab_to_nwb/markowitz_gillis_nature_2023_HEK/hekinterface.py

```diff
@@ -156,15 +156,15 @@ def add_to_nwbfile(self, nwbfile: NWBFile, metadata: dict) -> None:
         )
         signal_1p_series = OnePhotonSeries(
             name="Signal1PSeries",
-            data=H5DataIO(raw_signal, compression=True),
+            data=H5DataIO(raw_signal, compression=True, chunks=(14, 520, 688)),
```

Comment:
Are the image width/height consistent across all instances of data? Otherwise I'd recommend specifying this with extracted parameters so it generalizes

Reply:
They are consistent, but I went ahead and generalized anyway (might come in handy down the line).